zoomus/components/webinar.py

```diff
@@ -122,3 +122,34 @@ def remove_panelists(self, **kwargs):
         return self.delete_request(
             "/webinars/{}/panelists".format(kwargs.get("id")), params=kwargs
         )
+    
+    def participants_report(self, webinar_id):
+        url = f'/report/webinars/{webinar_id}/participants'
```

Comment:
follow the same syntax above:
def participants_report(self, **kwargs):
util.require_keys(kwargs, "id")

rename variable from webinar_id to id